internal/commands/secrets/create_inputs_test.go

```diff
@@ -0,0 +1,102 @@
+package secrets
+
+import (
+	"testing"
+
+	"github.com/10gen/realm-cli/internal/cli"
+	"github.com/10gen/realm-cli/internal/utils/test/assert"
+	"github.com/10gen/realm-cli/internal/utils/test/mock"
+	"github.com/Netflix/go-expect"
+)
+
+func TestCreateInputs(t *testing.T) {
+	for _, tc := range []struct {
+		description    string
+		createInputs   createInputs
+		prepareProfile func(p *cli.Profile)
+		procedure      func(c *expect.Console)
+		test           func(t *testing.T, i createInputs)
+	}{
+		{
+			description: "should prompt for secret name when not provided",
+			createInputs: createInputs{
+				Value: "value",
+			},
+			prepareProfile: func(p *cli.Profile) {},
+			procedure: func(c *expect.Console) {
+				c.ExpectString("Secret Name")
+				c.SendLine("name")
```

Comment:
im still a little muddy myself about the whole `ExpectEOF` stuff, but I'm surprised this test works here without it?  I'm gonna play around with this a little more locally

Reply:
Taking a look at this too. I can add it back in since it is probably best practice from what I see? Will play around with this and see what happens